docs/UserGuide.md

```diff
@@ -87,7 +86,9 @@ your own information. <br>
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines 
 as space characters surrounding line-breaks may be omitted when copied over to the application.
 
-</box>
```

Comment:
Is it your intention to remove the formatted box? It looks a bit plain now 😢 

Before:
<img width="529" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/e0c079d3-32e1-4287-80ab-d4df688775c7">

After:
<img width="530" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/2ea5479c-5e8a-4467-84c9-6e55bcaeb6f6">

Reply:
We were thinking it looks a bit strange for an entire section to be inside a formatted box. Can KIV for future discussions.